config/locales/user_mailer/en.yml

```diff
@@ -44,6 +44,17 @@ en:
         %{help_link} or %{contact_link}.
       intro: The email address for your %{app} account has been changed.
     help_link_text: Help Center
+    letter_expired:
+      help: ''
+      info: The letter we sent to your address has a code that expired and can no
```

Comment:
`info: The letter we mailed your address has a confirmation code that has expired and can no longer be used to complete the verification process. To verify your address and complete the identity verification process you will need to sign into %{link} and request another letter.`

Reply:
Trying to use "mail" as a verb to reinforce that this is coming via postal mail.